src/streaming/models/VideoModel.js

```diff
@@ -36,13 +36,16 @@ import Debug from '../../core/Debug';
 import Constants from '../constants/Constants';
 import Settings from '../../core/Settings';
 
+const READY_STATES_TO_EVENT_NAMES = function () {
```

Comment:
What's the reason this is an IIFE?

Reply:
More like how the Map Constructor works IMO, didn't like having a bunch of assigns at the top level

Reply:
Ohh is dynamically resolved keys an ES6 feature?

Reply:
I don't know but it didn't work when I tried it